argo-cd/go-gator/application.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: go-gator-app
+  namespace: go-gator
+spec:
+  project: default
+  source:
```

Comment:
are you sure this will work? where is a declaration of git project and folder which it will watch?

Reply:
Created file with main application, which launches 3 apps which are in `argo-cd/apps` folder